EIPS/eip-2981.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 2981
 title: ERC-721 Royalty Standard
```

Comment:
Perhaps **NFT Royalty Standard**  so as to remove the 721 specificity?

Reply:
I would not use NFT as ERC1155 allows tokens to not be NFT

Reply:
The goal is for NFT creators to receive royalties, correct? Saying ERC-721 in the title seems overly specific since 1155 should be supported as well. But **ERC-721 / ERC-1155 Royalty Standard** would be doubling down on specificity because a new NFT EIP would not be indicated. The fact that 1155 is technically semi-fungible seems less important than clarity in the title, IMHO.

Reply:
I agree that we want to widen the scope.  Leading with ERC721 is both restrictive and misleading.  I'm torn between something as wide as  "Simple Royalty Standard" and something that is a bit more specific about what problem this solves: "Royalty signaling Standard".  That leaves it open for a "Royalty payment standard" etc.